### DISCUSSION TITLE
Why do item textures not show in game?

### DISCUSSION THREAD

**Discussion by JoReSmi** [bridge-core/bridge.#335] [Add-On Q&A] [answered]
Why do item textures not show in game? I followed multiple tutorials and redid my item several times, but it wont show the texture. Can anyone help me? I don't have discord.

**Comment by Quasar500** [comment]
Did you activate the resource pack that has the textures in game?

**Reply by JoReSmi** [reply]
Yep

**Reply by Quasar500** [reply]
I pretty new to bridge but mabe I can help, can you give screenshots of all the items code and everything

**Comment by Eldyn** [comment, marked as answer]
1. did you add the minecraft:icon component to your item? 2. do not input 1.16.200 in your format version, bridge lets you input it but the item won't work. 3. did you enable the Holiday Features? without them, you cannot see your items in game. Hope this Helps !

**Reply by JoReSmi** [reply]
Hey thank you for the advice! Havent tried it yet, dont have time today, will do it tomorrow. One question though, should i format it for 1.16 or 1.16.100?

**Reply by Quasar500** [reply]
I just got this problem as well, I did these and it still doesn't work, my item just shows up as a black square

**Reply by Chikorita-Lover** [reply]
JoReSmi, use 1.16.100
Quasar, could you open a new discussion, mention me, and send the item behavior and `item_texture.json` files?

**Reply by Landosumm** [reply]
Thank you, **Eldyn**! I spent **_way_** too much time trying to figure out why my custom sword acted like an invisible item!
I enabled **Holiday Creator Features**, and it worked **_exactly_** how I wanted it.
